Thanks, I compiled it for arm64, but wxWidgets/samples/widgets won't:
```
root@cb0f2559de4e:/build/wxWidgets/samples/widgets# make
aarch64-w64-mingw32-windres -i../sample.rc -owidgets_sample_rc.o    --define __WXMSW__        --include-dir . --define WXUSINGDLL --define wxUSE_DPI_AWARE_MANIFEST=2 --include-dir ./../../samples --include-dir /build/wxWidgets/lib/wx/include/aarch64-w64-mingw32-msw-unicode-3.1 --include-dir ../../include
llvm-rc: Error in 24 statement (ID 1): 
Is a directory
aarch64-w64-mingw32-windres: error: llvm-rc failed
Makefile:319: recipe for target 'widgets_sample_rc.o' failed
make: *** [widgets_sample_rc.o] Error 1

root@cb0f2559de4e:/build/wxWidgets/samples/widgets# aarch64-w64-mingw32-windres -i../../samples/sample.rc -oaccesstest_sample_rc.o    --define __WXMSW__        --include-dir . --define WXUSINGDLL --define wxUSE_DPI_AWARE_MANIFEST=2 --include-dir ./../../samples --include-dir /build/wxWidgets/lib/wx/include/aarch64-w64-mingw32-msw-unicode-3.1 --include-dir ../../include -v
aarch64-w64-mingw32-clang -E -xc -DRC_INVOKED=1 -D__WXMSW__ -DWXUSINGDLL -DwxUSE_DPI_AWARE_MANIFEST=2 -I. -I./../../samples -I/build/wxWidgets/lib/wx/include/aarch64-w64-mingw32-msw-unicode-3.1 -I../../include ../../samples/sample.rc -o accesstest_sample_rc.o.preproc.rc 
llvm-rc -I . -I ./../../samples -I /build/wxWidgets/lib/wx/include/aarch64-w64-mingw32-msw-unicode-3.1 -I ../../include -I ../../samples accesstest_sample_rc.o.preproc.rc -c 1252 -fo accesstest_sample_rc.o.out.res 
llvm-rc: Error in 24 statement (ID 1): 
Is a directory
aarch64-w64-mingw32-windres: error: llvm-rc failed
root@cb0f2559de4e:/build/wxWidgets/samples/widgets# llvm-rc -I . -I ./../../samples -I /build/wxWidgets/lib/wx/include/aarch64-w64-mingw32-msw-unicode-3.1 -I ../../include -I ../../samples accesstest_sample_rc.o.preproc.rc -c 1252 -fo accesstest_sample_rc.o.out.res -v
Identifier: aaaaaaaa
Identifier: ICON
String: "sample.ico"
Identifier: sample
Identifier: ICON
...
Icon (wxICON_SMALL_REMOVEABLE): "wx/msw/removble.ico"
Bitmap (csquery): "wx/msw/csquery.bmp"
Bitmap (wxBITMAP_STD_COLOURS): "wx/msw/colours.bmp"
User-defined (type: 24, name: 1): "wx/msw/"
llvm-rc: Error in 24 statement (ID 1): 
Is a directory

root@cb0f2559de4e:/build/wxWidgets# grep -R "wx/msw/\""
samples/widgets/accesstest_sample_rc.o.preproc.rc:1 24 "wx/msw/" "arm64" "_dpi_aware_pmv2.manifest"
include/wx/msw/wx.rc:    #define wxMANIFEST_FILE "wx/msw/" wxRC_STR(wxMANIFEST_CPU) wxRC_STR(wxMANIFEST_DPI)
```

--- a/.github/workflows/ci_msw_cross.yml
+++ b/.github/workflows/ci_msw_cross.yml
@@ -86,7 +86,7 @@ jobs:
         run: |
           export DEBIAN_FRONTEND=noninteractive
 
-          packages="git make python3 python3-pip wine x11-xserver-utils xvfb"
+          packages="git make python3 python3-pip winbind wine x11-xserver-utils xvfb"
 
           case "${HOST_TRIPLET}" in
             x86_64-w64-mingw32)

--- a/.github/workflows/ci_msw_cross.yml
+++ b/.github/workflows/ci_msw_cross.yml
@@ -91,12 +91,14 @@ jobs:
           case "${HOST_TRIPLET}" in
             x86_64-w64-mingw32)
               packages="$packages g++-mingw-w64-x86-64 wine64"
+              winerun=wine64
               ;;
 
             i686-w64-mingw32)
               sudo dpkg --add-architecture i386
               sudo apt-get -q -o=Dpkg::Use-Pty=0 update
               packages="$packages g++-mingw-w64-i686 wine32"
+              winerun=wine
               ;;
 
             *)
@@ -106,6 +108,8 @@ jobs:
           esac
 
           sudo apt-get -qq install $packages
+
+          echo "WINERUN=${winerun}" >> $GITHUB_ENV
 
       - name: Checkout
         uses: actions/checkout@v2
@@ -164,7 +168,7 @@ jobs:
 
           httpbin_launch
 
-          wine ./test || rc=$?
+          $WINERUN ./test || rc=$?
 
           if [ -n "$rc" ]; then
             httpbin_show_log
@@ -199,4 +203,4 @@ jobs:
               sleep 3
           done
           echo 'Launching the GUI test:'
-          DISPLAY=:10 wine ./test_gui
+          DISPLAY=:10 $WINERUN ./test_gui

--- a/build/tools/httpbin.sh
+++ b/build/tools/httpbin.sh
@@ -16,7 +16,9 @@ httpbin_launch() {
 
     case "$(uname -s)" in
         Linux)
-            dist_codename=$(lsb_release --codename --short)
+            if command -v lsb_release > /dev/null; then
+                dist_codename=$(lsb_release --codename --short)
+            fi
             ;;
 
         Darwin)

